app/src/main/java/com/simply/birthdayapp/auth/signIn/presentation/SignInViewModel.kt

```diff
@@ -9,6 +10,11 @@ class SignInViewModel(
     private val setSignedInUseCase: SetSignedInUseCase,
 ) : ViewModel() {
 
+    val emailText = mutableStateOf("")
+    val passwordText = mutableStateOf("")
+    val emailError = mutableStateOf<String?>("")
+    val passwordError = mutableStateOf<String?>("")
```

Comment:
make member variables and public variables for public flows like:

private val _name = MutableStateFlow("")
val name = _name.asStateFlow()